Dockerfile-composer

```diff
@@ -0,0 +1,12 @@
+FROM php:7.0-cli as my-php
+
+RUN curl -sS https://getcomposer.org/installer | \
+    php -- --install-dir=/usr/bin/ --filename=composer \
+    && apt-get update \
+    && apt-get install -y zlib1g-dev \
+    && docker-php-ext-install zip
+
+WORKDIR /
+COPY . .
```

Comment:
Correct me if I'm wrong, but it looks like the source code is being baked into the Docker images (for composer and yarn). If so, it might be better to not include the source code in the image itself, but instead when running the tests, mount the source code directory in the docker run command (using the *-v* option).

This way, the tests are always run against fresh source code. Also, then the docker image doesn't have to be rebuilt every time the source code changes.

If you do it this way, you might want to also add an entrypoint that checks to make sure the source code directory is mounted. An example of this is in https://github.com/sugarcrm/eng-tools/tree/master/voodoo_driver. The entry_point.sh file does the mountpoint check, and the Dockerfile shows how to call entry_point.sh.